data/cy/census_individual_gb_wls.json

```diff
@@ -1815,37 +1860,37 @@
                                         "answers": [
                                             {
                                                 "id": "other-address-answer-building",
-                                                "label": "Enw neu rif y t\u0177",
+                                                "label": "House name or number",
```

Comment:
It does seem like we're dropping translations here that we have..has something changed with the content?

Reply:
Not that I can see, and it's in the .po file with the correct matching context. I don't know why it didn't insert it into the schema.

Reply:
Ah, got it. Yes, at least in some cases- e.g. a change of question from "One year ago, what was your usual address?" to "Enter details of their address one year ago.", which breaks the connection.